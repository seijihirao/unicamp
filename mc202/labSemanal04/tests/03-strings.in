push             ...   No fim tu hás de ver que as coisas mais leves são as únicas que o vento não conseguiu levar   ...
is-empty
print-last
print-first          
push um dia desses choverá   
is-empty
print-last
print-first
push     Talvez amanhã ou talvez num amanhã, mas choverá!   
is-empty
print-last
print-first
is-empty
exit
