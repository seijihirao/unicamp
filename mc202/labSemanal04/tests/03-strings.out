nope
...   No fim tu hás de ver que as coisas mais leves são as únicas que o vento não conseguiu levar   ...
...   No fim tu hás de ver que as coisas mais leves são as únicas que o vento não conseguiu levar   ...
nope
um dia desses choverá
...   No fim tu hás de ver que as coisas mais leves são as únicas que o vento não conseguiu levar   ...
nope
Talvez amanhã ou talvez num amanhã, mas choverá!
...   No fim tu hás de ver que as coisas mais leves são as únicas que o vento não conseguiu levar   ...
nope
